### DISCUSSION TITLE
How do I turn off pre-service information in webpack5？

### DISCUSSION THREAD

**Discussion by yanghuanrong** [webpack/webpack#14275] [Q&A]
<i> [webpack-dev-server] Project is running at: 
<i> [webpack-dev-server] Loopback: http://localhost:9000/
<i> [webpack-dev-server] On Your Network (IPv4): http://197.158.164.104:9000/
<i> [webpack-dev-server] On Your Network (IPv6): http://[fe80::1]:9000/
<i> [webpack-dev-server] Content not from webpack is served from '/path/to/public' directory

I've tried, `
client: {
logging: 'info',
}
` But you don't get the results you want

**Comment by alexander-akait** [comment]
Using this https://webpack.js.org/configuration/other-options/#level

**Reply by yanghuanrong** [reply]
Thank you! The problem was effectively solved
